js/src/BrushSelector.ts

```diff
@@ -412,6 +413,7 @@ export class BrushIntervalSelector extends BrushMixinXSelector {
                 (a: number, b: number) => a - b);
             this.update_mark_selected(pixel_extent);
         }
+        this.syncModelToBrush();
```

Comment:
This is the reason why we had the bounce. There is a first empty_selection called when we start brushing which was causing the bouncing. In #1098, I set the `brushing` attribute to `true` earlier so that it does not cause bouncing.